### PR TITLE
[JIT] fix googlenet no aux logits

### PR DESCRIPTION
**Pull request by eellison**
Fix for https://github.com/pytorch/vision/issues/1936. Also removed the constants because they were generating a warning and are no longer needed.

### CODE REVIEW

**Comment by mikeseven** [comment]
Unfortunately, the problem is still there with pretrained googlenet on torchvision 0.7.0, pytorch 1.6
The solution I found is to add 
if submodule is not none
on line 2234 in torch/jit/__init__.py so these bad aux variables don't crash scripting.

**Comment by fmassa** [comment]
@mikeseven could you send a PR fixing this?
